modules/tasks/tasks/src/main/resources/webapp/partials/recent-task-activity.html

```diff
@@ -2,13 +2,21 @@
 <div>
     <div class="box-content clearfix">
         <table id="table-activity" class="table table-striped table-lightblue table-recent-activity table-condensed centered">
-            <tbody ng-show="activities.length == 0">
-            <tr>
+            <tbody>
+            <tr ng-show="activities.length == 0">
                 <td>{{msg('task.info.notFoundActivities')}}</td>
             </tr>
-            </tbody>
-            <tbody ng-hide="activities.length == 0">
-            <tr ng-repeat="activity in activities">
+            <tr ng-show="logsDisabled()">
+                <td colspan="5" >
+                    <a class="activity-warn" href="#/tasks/settings">{{msg('task.info.logsDisabled')}}</a>
+                </td>
+            </tr>
+            <tr ng-show="successLogsDisabled()">
+                <td colspan="5">
+                    <a class="activity-warn" href="#/tasks/settings">{{msg('task.info.successLogsDisabled')}}</a>
+                </td>
+            </tr>
+            <tr ng-repeat="activity in activities" ng-hide="activities.length == 0">
```

Comment:
I think that activities could be in another "tbody". To clarify one tbody will handle all informations and another with activities.